docs/APIReference-Store.md

```diff
@@ -62,3 +68,36 @@ var mutation = new MyMutation({...});
 
 Relay.Store.update(mutation, {onFailure, onSuccess});
 ```
+
+### applyUpdate (static method)
+
+```
+static applyUpdate(mutation: RelayMutation, callbacks: {
+  onFailure?: (transaction: Transaction) => void;
```

Comment:
nit: `transaction: RelayMutationTransaction` (also should update the above doc for `update` for consistency)